MyRoutine/MyRoutine/Scenes/Diary/CellSetUp.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CellSetUp.swift
+//  MyRoutine
+//
+//  Created by Bùi Xuân Huy on 8/3/19.
+//  Copyright © 2019 huy. All rights reserved.
+//
+
+class CellSetUp: UITableViewCell {
+    
+    // MARK: - Outlets
+    @IBOutlet weak var lblState: UILabel!
```

Comment:
```suggestion
    @IBOutlet weak var stateLabel: UILabel!
```
tương tự với các cái bên dưới